suite-native/device/src/selectors.ts

```diff
@@ -72,3 +74,24 @@ export const selectDeviceTotalFiatBalanceNative = memoizeWithArgs(
         size: 20,
     },
 );
+
+// Unique symbols for all accounts that are on view only devices (excluding portfolio tracker)
+export const selectViewOnlyDevicesAccountsNetworkSymbols = memoize(
+    (state: DeviceRootState & AccountsRootState) => {
+        const rememberedDevices = selectDevices(state).filter(
+            d => d.remember && d.id !== PORTFOLIO_TRACKER_DEVICE_ID && d.state,
+        );
+
+        return Array.from(
```

Comment:
I think this could be simplified using ts-belt.

Something like this:
```tsx
pipe(
  selectDevices(state),
  A.filter(
            d => d.remember && d.id !== PORTFOLIO_TRACKER_DEVICE_ID && d.state,
        ),
  A.map(selectAccountsByDeviceState(state, d.state!),
  A.filter(a => a.visible)
  A.map(a => a.symbol)
  A.flat,
  A.unique,
)
```

Reply:
fab022f7fb